### ISSUE TITLE
Small bug

### ISSUE THREAD

**Issue by 833M0L3** [samir-dahal/sambat-widget-landing#12]
Thanks for adding the taskbar integration feature. Though this is not an official way to do it, it works with bugs that are bearable.

Leaving that aside, the bug that I am facing and which is really annoying right now is, whenever I accidentally click on the calendar sitting on the taskbar, it pops up a mini calendar and the calendar moves out of the taskbar. Could you add an option to disable that mini calendar pop-up when clicked on it so that the calendar doesn't move out of the tasbar? It's really annoying.

https://github.com/samir-dahal/sambat-widget-landing/assets/59522309/bfb77f37-f062-43e6-9393-1267a9f8fcf0

Also, could you add an option to only show the BS date? The AD date in the taskbar is taking up a lot of space, making the widget really long.

**Comment by samir-dahal** [comment]
For the fixed position, did you try out the latest version? Also, did you check the setting in general section about working area? Try play around with it and see if that works.

And for the AD date, I'll keep note of that.
Thanks.

**Comment by 833M0L3** [comment]
> For the fixed position, did you try out the latest version? Also, did you check the setting in general section about working area? Try play around with it and see if that works.

Yes I tried playing around with everything. I also tried force locking it's position , but it always goes outside the taskbar area whenever mini calendar appears once clicked on it.

I am using the latest version.
